addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2942,12 +2942,35 @@ var SnippetsMenu = Widget.extend({
                 break;
         }
 
-        const customizeBlock = $('<WE-CUSTOMIZEBLOCK-OPTIONS id="o_we_jw_toolbar_container"/>');
+        const customizeBlock = $('<WE-CUSTOMIZEBLOCK-OPTIONS id="o_we_editor_toolbar_container"/>');
         const $title = $("<we-title><span>" + titleText + "</span></we-title>");
 
         customizeBlock.append($title);
         customizeBlock.append(this.options.wysiwyg.toolbar.$el);
         $(this.customizePanel).append(customizeBlock);
+
+        $(document.body).on('click', this._checkEditorToolbarVisibility.bind(this));
+
+    },
+    /**
+     * Update editor UI visibility based on the current range
+     *
+     * hide the editor toolbar if the current selection range is out of the editable zone
```

Comment:
I don't think this comment is necessary. If you think it is, can youd add Maj to `Hide` and a "." at the end of the sentence?